doc/developers/contributing.rst

```diff
@@ -942,6 +942,11 @@ Optional Arguments
 In iterative algorithms, the number of iterations should be specified by
 an integer called ``n_iter``.
 
+In some estimators such as those in :mod:`sklearn.svm` and 
```

Comment:
You should add a separate header using the same formatting as the headers above.
Also, maybe add a sentence that any estimator that uses ``X`` with shape ``(n_samples, n_samples)``. needs to have this. Please also use ``(n_samples, n_samples)`` instead of ``[N, N]``